.github/workflows/on_test_update_gh_flaky_test.yml

```diff
@@ -1,19 +1,19 @@
-name: Update Jira Flaky
+name: Update Flaky Test Issue
 on:
    workflow_run:
      workflows: [Test]
      types: [completed]
 jobs:
-   jiraForFlakyTest:
-      if: ${{ github.event.workflow_run.conclusion == 'success' }}
-      name: Flaky Test To Jira
+   issueForFlakyTest:
+      if: github.event.workflow_run.conclusion == 'success'
```

Comment:
Does this mean that we only report Flaky tests when all tests eventually pass? Looking at the test run on this PR I see " 33640 tests run, 183 skipped, 6 failed. " and the check is marked as :red_circle:. I would hope that we report Flaky tests whenever the test run finishes, even if failures exist.

Reply:
> Does this mean that we only report Flaky tests when all tests eventually pass? 

No, we run mvn with `maven.test.failure.ignore=true`, so the workflow succeeded even with tests failure. It only fails if something else goes really wrong, i.e. mvn itself crashes